examples/features/load_balancing/client/main.go

```diff
@@ -123,4 +141,87 @@ func (*exampleResolver) Close()                                  {}
 
 func init() {
 	resolver.Register(&exampleResolverBuilder{})
+	balancer.Register(&customRoundRobinBuilder{})
+}
+
+const customRRName = "custom_round_robin"
+
+type customRRConfig struct {
+	serviceconfig.LoadBalancingConfig `json:"-"`
+
+	// N represents how often pick iterations chose the second SubConn
+	// in the list. Defaults to 3.
+	N uint32 `json:"n,omitempty"`
+}
+
+func (customRoundRobinBuilder) ParseConfig(s json.RawMessage) (serviceconfig.LoadBalancingConfig, error) {
+	lbConfig := &customRRConfig{
+		N: 2,
+	}
+	if err := json.Unmarshal(s, lbConfig); err != nil {
+		return nil, fmt.Errorf("custom-round-robin: unable to unmarshal customRRConfig: %v", err)
+	}
+	return lbConfig, nil
+}
+
+type customRoundRobinBuilder struct{}
+
+func (customRoundRobinBuilder) Name() string {
+	return customRRName
+}
+
+func (customRoundRobinBuilder) Build(cc balancer.ClientConn, bOpts balancer.BuildOptions) balancer.Balancer {
+	crr := &customRoundRobin{}
+	baseBuilder := base.NewBalancerBuilder(customRRName, crr, base.Config{HealthCheck: true})
+	crr.Balancer = baseBuilder.Build(cc, bOpts)
```

Comment:
I'm really not a fan of the base balancer...are we sure we want users using it as a template for how to do their own LB policy?  I think it would be better to show the actual connection management APIs here instead.

Reply:
Switched to petiole policy wrapping pick first after offline discussion.